frontend/src/app/api/tests/[testId]/finish-test/route.ts

```
@@ -0,0 +1,104 @@
+    if (!userEmail) {
+      return NextResponse.json(
+        { error: "Unauthorized" },
+        { status: 401 }
+      );
+    }
+
+    const xata = getXataClient();
+    
+    // Validate test ID
+    const testId = parseInt(params.testId);
+    if (isNaN(testId) || testId <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)